### DISCUSSION TITLE
pacx view add clean

### DISCUSSION THREAD

**Discussion by goncalosilvalaranjeira** [neronotte/Greg.Xrm.Command#88] [Ideas]
Hi Riccardo Gregori,
I was wondering if it would make sense to add a command that creates a new view based on an existing view, but without any filters applied. This command would create an 'All {records}' view, displaying the full dataset without any filtering.
I'm happy to provide any help you may need

**Comment by neronotte** [comment]
Hi Gonçalo,
done. Wait for the nex pacx release 😉
